src/patterns/builder/Car.java

```diff
@@ -0,0 +1,64 @@
+package patterns.builder;
+
+public class Car {
+
+   private String engine;
+   private String wheelType;
+   private String transmission;
+   private String brakes;
+
+    public String getEngine() {
+        return engine;
+    }
+
+    public String getWheelType() {
+        return wheelType;
+    }
+
+    public String getTransmission() {
+        return transmission;
+    }
+
+    public String getBrakes() {
+        return brakes;
+    }
+
+    public static class Builder {
+        private Car newCar;
+
+        public Builder() {
+            newCar = new Car();
```

Comment:
creation of a car should be done in `build` method, because it is a logical point of creation an object you need and set all required fields